packages/pages/src/vite-plugin/build/build.ts

```diff
@@ -150,16 +137,15 @@ const discoverRenderTemplates = (
 
   // Move the [compiled] _server.ts and _client.ts render template to /assets/render
   const clientServerRenderTemplates = getGlobalClientServerRenderTemplates(
-    projectStructure.templatesRoot,
-    projectStructure.scopedTemplatesPath
+    projectStructure.getTemplatePaths()
   );
 
   // server
-  entryPoints["render/_server"] =
+  entryPoints[`${projectStructure.config.subfolders.renderBundle}/_server`] =
```

Comment:
you could destructure here as well to make things easier to read.

Reply:
What do you mean by destructure here?